Experiments/Experiments/FeatureFlag.swift

```diff
@@ -54,6 +54,10 @@ public enum FeatureFlag: Int {
     ///
     case orderCreation
 
+    /// Allows new orders to be manually created
```

Comment:
May be worth mentioning that this is for creating/syncing draft orders, e.g. "Allows new orders to be created and synced as drafts"

Reply:
I totally forgot to update the documentation 🤦